### DISCUSSION TITLE
難以定義元宇宙   Meta陷入困境和瓶頸

### DISCUSSION THREAD

**Discussion by HuangHsinTzu** [1101-GS4719-C/Python-Community-GS4719-C#1040] [General]
前陣子很夯的元宇宙的發展好像不如預期
文章提到最主要的問題是每有整合再一起
不同公司推出不同的產品和世界
還有就是想要體驗完整的元宇宙需要很昂貴且笨重的儀器
一般家庭根本沒有能力
其實看到這篇新聞時有點小小開心
因為之前看到元宇宙的概念出來後
我就覺得在真實世界生活已經夠有趣夠忙了
要同時在兩個世界裡經營自己聽起來就超累
話說，
終於知道為什麼FB和IG的廣告越來越多了哈哈哈
https://www.technice.com.tw/web30/metaverse/33078/

**Comment by manipulateed** [comment]
我覺得我們的首要任務，是要盡可能地解釋我們對這些技術的願景是什麼，以及我們認為隨著技術的發展需要考慮的挑戰，並讓我們正在做的一切開放且透明。

**Comment by dabbie109303501** [comment]
首先 要讓民眾熟悉並習慣元宇宙並不容易
再者 目前的技術根本就不完善 虛擬人物的動作 相貌 還很粗糙
以及很多公司都想分元宇宙的一杯羹 有各式各樣發展不完善的元宇宙
難怪會讓民眾感到混亂
